connect/api/src/main/java/org/apache/kafka/connect/data/SchemaProjector.java

```diff
@@ -109,16 +109,14 @@ private static Object projectStruct(Schema source, Struct sourceStruct, Schema t
                     Object targetFieldValue = project(sourceField.schema(), sourceFieldValue, targetField.schema());
                     targetStruct.put(fieldName, targetFieldValue);
                 } catch (SchemaProjectorException e) {
-                    throw new SchemaProjectorException("Error projecting " + sourceField.name(), e);
+                    throw new SchemaProjectorException("Error projecting " + fieldName, e);
```

Comment:
That's where I'm referring to: 
`String fieldName = targetField.name();`

but I didn't follow too many levels of redirection ...